CHANGELOG.md

```diff
@@ -2,155 +2,188 @@
 
 All notable changes to this project will be documented in this file.
 
-The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/),
```

Comment:
It appears to me that we are staying extremely close to the Keep A Changelog format, and it seems to me we are still following all the most important conventions (e.g. an "Unreleased" section at the top), but only changing two details (what headers under each release, and inline vs footnote-style links) -- does that match your understanding?

If so, would we consider keeping the link and saying something like:

```
The format is inspired by [Keep a Changelog](https://keepachangelog.com/en/1.0.0/), but has minor changes to the headers we use to breakout changes in each release.
```